lang/perl/patches/310-recompile-hack.patch

```diff
@@ -5,12 +5,12 @@ patch has been reported to work around that issue.
 
 --- a/Makefile.SH
 +++ b/Makefile.SH
-@@ -818,7 +818,7 @@ $(LIBPERL): $& $(obj) $(DYNALOADER) $(LI
+@@ -867,7 +867,7 @@ $(LIBPERL): $& $(perllib_objs) $(DYNALOA
  	true)
  		$spitshell >>$Makefile <<'!NO!SUBS!'
  	rm -f $@
--	$(LD) -o $@ $(SHRPLDFLAGS) $(obj) $(DYNALOADER) $(libs)
-+	$(CC) -o $@ $(SHRPLDFLAGS) $(obj) $(DYNALOADER) $(libs)
+-	$(LD) -o $@ $(SHRPLDFLAGS) $(perllib_objs) $(DYNALOADER) $(libs)
++	$(CC) -o $@ $(SHRPLDFLAGS) $(perllib_objs) $(DYNALOADER) $(libs)
```

Comment:
Seems like there is no difference.  I ran with and without the 310 patch after running

`make package/feeds/packages/perl/clean`

[lang_perl_compile_patch310.txt](https://github.com/openwrt/packages/files/988399/lang_perl_compile_patch310.txt)
[lang_perl_compile_NOpatch310.txt](https://github.com/openwrt/packages/files/988398/lang_perl_compile_NOpatch310.txt)